README.md

```diff
@@ -2,7 +2,8 @@
 
 # FL4Health
 
-Principally, this repository contains the federated learning (FL) engine aimed at facilitating FL research, experimentation, and exploration, specifically targetting health applications.
+Principally, this repository contains the federated learning (FL) engine aimed at facilitating FL research, experimentation, and exploration, with a specific focus on health applications.
+<!-- TOC -->
 <!-- TOC -->
 
 - [FL4Health](#fl4health)
```

Comment:
nit: I really like the table of contents, but its rather long because all of the components are linked. How about we just link to the Components header, since each subheader is less than page, its easy enough to navigate by scrolling

Reply:
That's fair. This is the "autogenerated TOC" from vs code. I'll pare it down manually.